js/views/modals/Settings/Page.js

```diff
@@ -30,9 +30,54 @@ export default class extends baseVw {
       'click .js-avatarRight': 'avatarRightClick',
       'click .js-headerLeft': 'headerLeftClick',
       'click .js-headerRight': 'headerRightClick',
+      'change .js-colorPicker': 'handleColorChosen',
+      'change .js-colorCode': 'handleColorCodeEntered',
     };
   }
 
+  /** Handles when a hex color code is entered by updating color picker. */
+  handleColorCodeEntered(event) {
+    const colorPickerId = $(event.target).data('color-picker-id');
+    const $colorPicker = this.getCachedElement(colorPickerId);
+    const newHexColorCode = event.target.value;
+
+    // If the text passes a basic RegExp for a valid 6 digit hex value,
+    // update the color picker's color.
+    if (/^#([0-9a-f]{6})$/i.test(newHexColorCode)) {
+      $colorPicker.val(newHexColorCode);
+    }
+  }
+
+  /** Handles when a color is chosen from the color picker by updating hex color code text. */
+  handleColorChosen(event) {
+    const hexInputId = $(event.target).data('hex-input-id');
+    const $hexInput = this.getCachedElement(hexInputId);
+    const newColor = event.target.value;
+
+    $hexInput.val(newColor);
+  }
+
+  /** Returns a given jquery Element or a locally cached version in this view for optimization. */
+  getCachedElement(elementId) {
```

Comment:
Great feedback. 
- The map is now cleared in render, but the entire <getCachedElement(selector)> method now resides in baseVw. The clearing of this cache is done in subviews with a super.render() at the beginning of their render methods. 
- Selector is now used rather than Id. I like that idea.